ignition-server/controllers/local_ignitionprovider.go

```diff
@@ -238,6 +238,22 @@ func (p *LocalIgnitionProvider) GetPayload(ctx context.Context, releaseImage str
 		return nil, fmt.Errorf("failed to extract templates from image: %w", err)
 	}
 
+	payloadVersion, err := semver.Parse(imageProvider.Version())
+	if err != nil {
+		return nil, fmt.Errorf("failed to parse payload version: %w", err)
+	}
+
+	// set the component to the correct binary name and file path based on the payload version
+	clusterConfigComponent := "cluster-config-api"
+	clusterConfigComponentShort := "cca"
+	clusterConfigFile := "usr/bin/render"
+
+	if payloadVersion.Minor < 15 {
```

Comment:
```suggestion
	if payloadVersion.Compare(semver.Version{Major: 4, Minor: 15}) < 0 {
```
Should be be more explicit like this?

Reply:
there shorthand functions for this. `LT()`, `LTE()`, `GT()`, etc.
I would prefer the comparison to be `less than or equal to 4.14` instead of `less than 4.15`. I think its more clear to the reader what versions is targeted this way, i.e.:
```golang
if payloadVersion.LTE(semver.Version{Major: 4, Minor: 14}) {
}
```

Reply:
Wouldn't LTE of 4.14 be wrong?
If we compared 4.14.7 to 4.14 (which has patch set to 0) LTE would give you false, wheres IIUC 4.14.7 still needs the other image. If that is right, it should be LT(semver.Version{Major:4, Minor: 15})

Reply:
Code is currently using LT

Reply:
@celebdor oh yeah you are right, do all 4.15.z version have the new image? if not we would need to specifically set the patch as well anyway.

Reply:
Yeah  I tink all versions of 4.15 should  be using the new image